prefect_sqlalchemy/credentials.py

```diff
@@ -110,17 +111,20 @@ class DatabaseCredentials:
             DBAPI's connect() method as additional keyword arguments.
     """
 
+    _block_type_name = "Database Credentials"
+    _logo_url = "https://github.com/PrefectHQ/orion/blob/main/docs/img/collections/sqlalchemy.png?raw=true"  # noqa
```

Comment:
This image wasn't loading for me in the UI. I found a generic database icon and uploaded it to our CMS.
```suggestion
    _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/3xLant5G70S4vJpmdWCYmr/bfec58128d4d985ff29c220c58529533/download.svg"  # noqa
```